tests/test_cli.py

```diff
@@ -83,3 +90,22 @@ def test_cli_almanack_check(tmp_path):
             "Software Gardening Almanack score:",
         ]
     )
+
+    # gather output and return code from running a CLI command
+    stdout, _, returncode = run_cli_command(
+        command=[
+            "almanack",
+            "check",
+            repo.path,
+            "--ignore",
+            "'SGA-GL-0002','SGA-GL-0026'",
```

Comment:
would there ever be a need to reference a code against all existing codes to throw a warning or error if the ignore key is incorrect?

Reply:
This is a great idea - and something missing in a lot of existing tools which follow a similar approach. I'll give a try at this and follow up with more changes.

Reply:
Added this in with a further commit.